### DISCUSSION TITLE
Is there a way to Archive to multiple harvester

### DISCUSSION THREAD

**Discussion by VercauterenK** [ericaltendorf/plotman#561] [Q&A]
Hi All,

Is there a way to archive to multiple pc's.
I have 1 pc that makes 42 plots every 7 hour.
Now he cant archive in time to 1 harvester.
Is there a way to split it to multiple harvester?

Thanks in advance,
Kevin

**Comment by cyperbg** [comment]
Sorry for the off topic, but do you mind sharing what hardware you use to make 42 plots every 7 hours. Cheers

**Reply by VercauterenK** [reply]
Copied from a other topic where i reacted about my total setup.
The threadripper makes 42 plots it also has 8 SSD (sata) 2 tb

Hi,

Thanks for the answer i changed my infrastructure.
Here is the image of the diagram that i made.
https://www.mobilrent.be/infra.png
everything is finally running after 4 days of setup and hardware placing

Plotter 1 -> Plotter 6 = CPU AMD 5900x
2 x 2TB pcie 4 nvme
1 x 1TB pcie 1 ssd
64 GB ram
1 x 10TB external HDD (ass buffer to archive to harvester)
Using plotman with archival

Plotter 7 / Harvester 4 = CPU AMD threadripper 3990x
3 x 2TB pcie 4 nvme
256 GB Ram
9 x 1TB pcie 3 ssd (can be maxed to 4x5)
13 x 10TB external HDD ( max is 13x7)
using plotman

Harvesters = CPU I5 processor
32 GB RAM
13 x 10TB external HDD ( max is 13x7)

Farmer = CPU AMD 5900x
64 GB ram

There are still 4 plotters and 2 harvester that are gonna be delivered this week. Will it be good to still use 1 farmer if i max out the HDD so 13x7 per harvester.

My farmer is running chia monitor and every harvester is slave chia monitor. Now i noticed when i open the chia monitor dash the average challenge go from 70 to 32 per 5 min.

All machines are using chia cli linux and i have 1 windows pc at home that is running my wallet to make transfers etc other pc are in my farming / magazin.

I am totally new to HDD mining but i am doiing gpu for 5 years and have around 500 rigs. Since gpu's are hard to get and i just bought my 3th warehouse i filling it up with HDD mining (it takes so few electrical power and place that i can make it huge).
There are 500 HDD on my way gonna be delivered in parts.

What do you think about the setup?
